docs/usage/configuration-options.md

```diff
@@ -2159,6 +2159,18 @@ Depending on its running schedule, Renovate may run a few times within that time
 
 Add to this object if you wish to define rules that apply only to major updates.
 
+## milestone
+
+If set to an existing and `open` milestone, the milestone will be added to the PR.
```

Comment:
```suggestion
If set to an existing and `open` milestone, the milestone will be added to the PR.
This will only be done once on PR creation.
```

I know users will be confused in future why PR's are not updated after config change 😉